CONTRIBUTING.md

```diff
@@ -17,9 +17,40 @@ Asumming that you have Node 12.10.x or newer and `npm` installed, install the re
 npm ci
 ```
 
-### Changesets
+### Pull request - Conventional Commits
```

Comment:
is there a pre-commit hook we can leverage to achieve this?

Reply:
Something like [this](https://dev.to/mbarzeev/a-git-hook-for-commit-messages-validation-no-husky-just-js-1hni) or [this](https://github.com/compilerla/conventional-pre-commit)

Reply:
I don't feel getting every commit into conventional commit standard would be beneficial, as for each change to main a PR is created, and commits are squashed, leaving the commit message in main as the PR title, therefore I feel checking the title would be best. There are some actions in Github for that kind of stuff. I'll add it so that we can check if it is worth the use

Reply:
@dubeyShivank do you feel the GH Action I added would be enough?

Reply:
let me check it out :)

Reply:
Now that I've seen the template, yeah should be good